service/src/main/java/greencity/service/CustomShoppingListItemServiceImpl.java

```diff
@@ -203,6 +201,15 @@ public CustomShoppingListItemResponseDto updateItemStatus(Long userId, Long item
             }.getType());
     }
 
+    @Override
+    public List<ShoppingListItemDto> findByActiveByUserIdAndLanguageCode(Long userId, String code) {
+        List<ShoppingListItemDto> shoppingListItems = new ArrayList<>();
+        customShoppingListItemRepo.findByActiveByUserIdAndLanguageCode(userId, code)
+            .forEach(x -> shoppingListItems.add(modelMapper.map(
```

Comment:
Would it better to use stream().map() instead of .foreach()? This way you will not need to use the new list

Reply:
Changed like this 
![image](https://user-images.githubusercontent.com/48826533/116546229-0ab73000-a8fa-11eb-9832-999e7f8987d2.png)